galaxy_api/api/models.py

```diff
@@ -35,9 +35,7 @@ class Namespace(models.Model):
     avatar_url = models.CharField(max_length=256, blank=True)
     description = models.CharField(max_length=256, blank=True)
     resources = models.TextField(blank=True)
-
-    # References
-    groups = models.ManyToManyField(auth_models.Group, related_name="namespaces")
+    groups = models.CharField(max_length=256, default="")
```

Comment:
If this field is intended to be used as a list of groups, please don't store it as a comma-separated string.
You can store it as denormalized list or as a JSON field.